addons/account/views/report_invoice.xml

```diff
@@ -109,12 +109,14 @@
                                 </tr>
                             </thead>
                             <tbody class="invoice_tbody">
+                                <t t-set="current_subtotal_tax_exl" t-value="0"/>
                                 <t t-set="current_subtotal" t-value="0"/>
                                 <t t-set="current_total" t-value="0"/>
                                 <t t-set="lines" t-value="o.invoice_line_ids.sorted(key=lambda l: (-l.sequence, l.date, l.move_name, -l.id), reverse=True)"/>
 
                                 <t t-foreach="lines" t-as="line">
-                                    <t t-set="current_subtotal" t-value="current_subtotal + line.price_subtotal"/>
+                                    <t t-set="current_subtotal_tax_exl" t-value="current_subtotal_tax_exl + line.price_subtotal"/>
```

Comment:
Can't you use the existing variables (`current_subtotal` & `current_total`) instead ?

Reply:
Not really as both current_subtotal and current_total are used for different things.